webapp/fds-bootstrap.js

```diff
@@ -54,6 +54,27 @@ if (!locale || locale === 'en-us') {
         }
         platformBrowserDynamic().bootstrapModule(FdsModule, {providers: providers});
     }).fail(function () {
-        platformBrowserDynamic().bootstrapModule(FdsModule, {providers: providers});
+        // was this a country specific locale? if so, try to get the generic version of the language
+        const localeTokens = locale.split('-');
+        if (localeTokens.length === 2) {
+            translationFile = './webapp/locale/messages.' + localeTokens[0] + '.xlf';
+            $.ajax({
+                url: translationFile,
+                dataType: 'text'
+            }).done(function (translations) {
+                // add providers if translation file for locale is loaded
+                if (translations) {
+                    providers.push({provide: TRANSLATIONS, useValue: translations});
+                    providers.push({provide: TRANSLATIONS_FORMAT, useValue: 'xlf'});
+                    providers.push({provide: LOCALE_ID, useValue: localeTokens[0]});
+                }
+                platformBrowserDynamic().bootstrapModule(FdsModule, {providers: providers});
+            }).fail(function () {
+                platformBrowserDynamic().bootstrapModule(FdsModule, {providers: providers});
```

Comment:
> nifi-fds@0.3.0-SNAPSHOT eslint
> eslint webapp platform


/Users/jaslan/Development/nifi-fds/target/frontend-working-directory/webapp/fds-bootstrap.js
  73:16  error  Block must not be padded by blank lines  padded-blocks